bootstrap/mocks/configs.go

```diff
@@ -84,22 +89,29 @@ func (crm *configRepositoryMock) RetrieveAll(key string, filter bootstrap.Filter
 		name = strings.ToLower(s)
 	}
 
+	var total uint64
 	for _, v := range crm.configs {
 		id, _ := strconv.ParseUint(v.MFThing, 10, 64)
-		if id >= first && id < last {
-			if (state == -1 || v.State == state) &&
-				(name == "" || strings.Index(strings.ToLower(v.Name), name) != -1) &&
-				v.Owner == key {
+		if (state == emptyState || v.State == state) &&
+			(name == "" || strings.Index(strings.ToLower(v.Name), name) != notFoundIdx) &&
```

Comment:
fix the tabulation

Reply:
It's the default way `gofmt` formats the code. I can only keep it inline, but I like to have these conditions separated because I find it a little more readable.